src/media/js/apps.js

```diff
@@ -147,7 +147,8 @@ define('apps',
 
             if (settings.gametimeEnabled &&
                 app.isWebsite &&
-                app.device_types.indexOf(capabilities.os.type) === -1) {
+                app.device_types.indexOf(capabilities.os.type) === -1 &&
+                app.keywords.indexOf('featured-game') !== -1) {
               // Don't show desktop website games on mobile.
```

Comment:
I think this comment is a bit weak. If this is disabling a button or used in a filter etc we should probably mention it.